src/pages/organize/[orgId]/people/[personId]/index.tsx

```diff
@@ -13,12 +13,16 @@ import SinglePersonLayout from 'layout/organize/SinglePersonLayout';
 import SnackbarContext from 'hooks/SnackbarContext';
 import { TagManagerSection } from 'components/organize/TagManager';
 import ZetkinQuery from 'components/ZetkinQuery';
-import { personResource, personTagsResource } from 'api/people';
+import {
+  personFieldsResource,
+  personResource,
+  personTagsResource,
+} from 'api/people';
 import { scaffold, ScaffoldedGetServerSideProps } from 'utils/next';
 
 export const scaffoldOptions = {
   authLevelRequired: 2,
-  localeScope: ['layout.organize', 'pages.people'],
+  localeScope: ['layout.organize', 'pages.people', 'misc.person'],
```

Comment:
I don't think this is necessary. The entire `misc` scope should be automatically included on all pages.